blog/Webots-2018-a-release.md

```diff
@@ -0,0 +1,58 @@
+# Version R2018a Released
+
+<p id="publish-data">By XXX - XXth XXX 2018</p>
+
+---
+
+Today we're happy to announce the release of Webots R2018a. This new release brings a ton of new features and improvements, coupled with the same dedicated patches addressing bugs and regressions. Listed below are some of the key features of this release (for a full list of changes please refer to the ChangeLog, found [here](https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html)).
+
+
+## Improved Automobile Simulation Integration
+
+The integration of the automobile simulation has been improved in Webots R2018a.
```

Comment:
You can remove the "the" before "automobile simulation".

Also, for greater clarity, I'd change this to "Automobile simulation tools have been largely integrated into the default Webots resources."